src/main/scala/ai/privado/languageEngine/php/processor/PhpProcessor.scala

```diff
@@ -89,7 +90,7 @@ class PhpProcessor(
       .withOutputPath(cpgOutput.toString)
       .withIgnoredFilesRegex(ruleCache.getExclusionRegex)
       .withPhpParserBin(PhpProcessor.parserBinPath)
-      .withDownloadDependencies(!config.skipDownloadDependencies)
+      .withDownloadDependencies(Try(!config.skipDownloadDependencies).getOrElse(false))
```

Comment:
Why do we need a Try here?

Reply:
For test cases where we want to download dependency or not we need to pass the respective flag in PrivadoInput, use `withPrivadoInput` while writing test case

Reply:
Tried, but reverted since we will have an issue where config is still not present in tests, but is expected still since it uses the same PhpProcessor now

Reply:
@tuxology If it is needed in a test case, it needs to passed there, this helps in understanding the config needed to make the test case work. Ensure we pass PrivadoConfig with the flag set to all such test cases